components/constellation/constellation.rs

```diff
@@ -2644,6 +2644,7 @@ where
             ipc::channel().expect("Failed to create IPC channel!");
         let (storage_ipc_sender, storage_ipc_receiver) =
             ipc::channel().expect("Failed to create IPC channel!");
+        let (mut _webgl_threads_sender, mut webgl_threads_receiver) = (None, None);
```

Comment:
You don't need the sender here, since it is only used inside the ` if let Some(webgl_threads)` block, so you can nest it to it, without wrapping it in an option.

Reply:
Indeed, I applied your suggestions